atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/DefaultCommitTimestampLoader.java

```diff
@@ -221,20 +219,19 @@ private void waitForCommitterToComplete(@Nullable TableReference tableRef, LongI
     }
 
     private Timer getTimer(String name) {
-        return metricsManager.registerOrGetTimer(CommitTimestampLoader.class, name);
+        return metricsManager.registerOrGetTimer(DefaultCommitTimestampLoader.class, name);
```

Comment:
Don't care about the metrics staying the same?

Reply:
Oof, good catch - yeah I'll revert that